invent-code-1/main.cpp

```diff
@@ -1,15 +1,27 @@
-#incldue <iostream>
+#include <iostream>
 #include <string>
 using namespace std;
 
 int main() {
+  int original;
+  float original_float;
+  original_float = 13.5;
 
+  cout << "Please enter an integer: ";
+  cin >> original;
+
+  int clone = original + 14;
+  int product = original * clone;
+  cout << "The product is: " << product << endl;
 }
 
 /*
 Table at the end of the program.
 
 Type   | Name     | Value
 ------ | -------- | -----
-       |          |
+int    | original | if = 2
+float  | original_float| 13.5
+int    | clone    | 16
```

Comment:
this actually rounds down to 15.5. Also should remove original_float since its not used anywhere else